jobs/build/refresh-images/Jenkinsfile

```diff
@@ -178,4 +179,24 @@ Jenkins job: ${env.BUILD_URL}
         }
 
     }
+
+    stage ('Attach Images') {
+        if (ADVISORY_ID != "") {
+            try {
+                buildlib.elliott """
+ --group 'openshift-${OSE_MAJOR}.${OSE_MINOR}'
+ advisory:find-builds
+ --kind image
+ --attach ${ADVISORY_ID}
+"""
+            } catch {
+                // Replace flow control with: https://jenkins.io/blog/2016/12/19/declarative-pipeline-beta/ when available
+                mail(to: "${MAIL_LIST_FAILURE}",
```

Comment:
Missing closing paren?

Reply:
Copy-paste fails me again! Fixed with also a link to the Jenkins job in the email.